### PR TITLE
Diabetes

### PR DESCRIPTION
**Pull request by Vayel**

### CODE REVIEW

**Comment by Vayel** [comment]
@MaxHalford we'd better make a release after it's merged as it fixes two bugs (ranking normalization and package data).

**Comment by MaxHalford** [comment]
The PR looks good. However now that we are versioning things, can we start maintaining a `CHANGELOG.md` file? A la [keepachangelog](https://keepachangelog.com/en/1.0.0).

File: setup.py [hunk 1]
```diff
@@ -12,13 +12,13 @@
...
-LONG_DESCRIPTION_CONTENT_TYPE = 'text/markdown'
-URL = 'https://github.com/MaxHalford/ethik'
-EMAIL = 'maxhalford25@gmail.com'
-AUTHOR = 'Max Halford'
-REQUIRES_PYTHON = '>=3.6.0'
+NAME = "ethik"
+DESCRIPTION = "A toolbox for fair and explainable machine learning"
+LONG_DESCRIPTION_CONTENT_TYPE = "text/markdown"
+URL = "https://github.com/MaxHalford/ethik"
+EMAIL = "maxhalford25@gmail.com"
```
Comment (MaxHalford) [inline setup.py]: You can add yourself to the list of authors, see [this example](https://stackoverflow.com/questions/9999829/how-to-specify-multiple-authors-emails-in-setup-py) :)

**Comment by MaxHalford** [comment]
I guess you can now add a `0.1.1` tag to this commit and push. If all goes well the new version should be pushed to PyPI.

**Pull request merged**